src/intermediate_public_item.rs

```diff
@@ -95,8 +104,543 @@ impl<'a> IntermediatePublicItem<'a> {
         }
         .to_owned()
     }
+
+    pub fn render_token_stream(&self) -> TokenStream {
```

Comment:
Related to my other comment: Instead of "duplicating" this function like this, shouldn't we make e.g. `IntermediatePublicItem::type_string_for_item()` and `IntermediatePublicItem::prefix()` produce `Token`s instead of strings?